dev/ci/README-developers.md

```diff
@@ -40,11 +40,39 @@ You can also run one CI target locally (using `make ci-somedev`).
 
 See also [`test-suite/README.md`](../../test-suite/README.md) for information about adding new tests to the test-suite.
 
+### Light and full CI
+
+By default, only a light CI is run, mostly testing only Coq itself.
+Before merging a PR, it must also pass full CI testing multiple third party
+developments. See [`CONTRIBUTING.md`](../../CONTRIBUTING.md#understanding-automatic-feedback)
+for more details.
+
 ### Breaking changes
 
-When your PR breaks an external project we test in our CI, you must
-prepare a patch (or ask someone—possibly the project author—to
-prepare a patch) to fix the project. There is experimental support for
+When your PR breaks external projects we test in our CI, you must
+assess the breakage. If it is a bug your PR introduces, it should be
+fixed. Otherwise, you must assess the impact of the breakage on a few
+CI entries at the bottom of its hierarchy, typically mathcomp, stdlib
+and stdpp. For legitimate breakages, for instance to remove something
+already deprecated, you must then ask the project maintainers to
+prepare a patch. For plugins, you must prepare yourself an overlay.
+Ultimately, for non-plugin CI projects, the responsibility of
+preparing a patch falls on the project maintainer, but PR writers are
```

Comment:
It is still ok to ask the project maintainer for help for plugins.

Reply:
Indeed, added a comment